apps/shell/bin/setup

```diff
@@ -50,17 +93,19 @@ chdir APP_ROOT do
   end
 
   puts "\n== Building Shell App =="
-  puts "DEFAULT_SSHHOST = #{DEFAULT_SSHHOST || OLD_SSHHOST || "localhost"}"
+  puts "DEFAULT_SSHHOST = #{DEFAULT_SSHHOST || OLD_SSHHOST}"
 
   puts "\n== Installing dependencies =="
   rm_rf "node_modules"
   sh "npm install"
 
-  if DEFAULT_SSHHOST && DEFAULT_SSHHOST != OLD_SSHHOST
+  if DEFAULT_SSHHOST != OLD_SSHHOST
     puts "\n== Setting default ssh host =="
     sh "echo 'DEFAULT_SSHHOST=#{DEFAULT_SSHHOST}' > .env"
   end
 
+  sh "echo 'SSHHOST_WHITELIST=#{whitelist}' >> .env"
```

Comment:
I'd get rid of this and the block above that writes DEFAULT_SSHHOST to .env. This was added a long time ago when we didn't have build by rpm and no configuration in /etc/